elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -116,7 +117,12 @@ public static Method findMethod(Class<?> entityClass, String name, Class<?>... p
     }
 
     protected EntityBinding getEntityBinding(Class<?> entityClass) {
-        return entityBindings.getOrDefault(lookupEntityClass(entityClass), EntityBinding.EMPTY_BINDING);
+        Class<?> entitySearched = isMappedInterface(entityClass) ? entityClass : lookupEntityClass(entityClass);
```

Comment:
Would the intent be clearer here if we explicitly return EntityBinding.EMPTY_BINDING if the interface is mapped?

```
if (isMappedInterface(entityClass) {
    return EntityBindings.EMPTY_BNDING;
}
```

Reply:
Yeah this is a subjective one. I actually had that at first, but after refactoring it a few times.   I thought this way ended up being more clear on "why we're doing this".  But I might be coming from a bad perspective.  So input would be appreciated.